Google.PowerShell/Compute/GceMetadataCmdlet.cs

```diff
@@ -117,7 +119,7 @@ public class GetGceMetadataCmdlet : GCloudCmdlet
         /// <summary>
         /// Make this a field, so it can be aborted if cmdlet stops e.g. the user hits Ctrl-C.
         /// </summary>
-        private HttpWebRequest _request;
+        private static HttpClient _client = new HttpClient();
```

Comment:
I think this should be disposed of at some point? Does this implement IDisposable?

Reply:
According to what I read, you don't have to: https://stackoverflow.com/questions/15705092/do-httpclient-and-httpclienthandler-have-to-be-disposed.

Reply:
I did not know this.